include/boost/geometry/iterators/point_reverse_iterator.hpp

```diff
@@ -54,7 +44,7 @@ class point_reverse_iterator
     template <typename OtherGeometry>
```

Comment:
In the private constructor above please rename `base` to `base_type`.
To avoid a long line, please write it as:
```
inline point_reverse_iterator(base_type const& base_it)
    : base_type(base_it)
{}
```